src/Console/HelperRegistry.php

```diff
@@ -92,14 +92,18 @@ protected function _throwMissingClassError(string $class, ?string $plugin): void
      *
      * Part of the template method for Cake\Core\ObjectRegistry::load()
      *
-     * @param string $class The classname to create.
+     * @param \Cake\Console\Helper|string $class The classname to create.
      * @param string $alias The alias of the helper.
      * @param array $config An array of settings to use for the helper.
      * @return \Cake\Console\Helper The constructed helper class.
      * @psalm-suppress MoreSpecificImplementedParamType
      */
-    protected function _create($class, string $alias, array $config): Helper
+    protected function _create(object|string $class, string $alias, array $config): Helper
```

Comment:
why not `Helper|string` ??

Reply:
contravariance
https://www.php.net/manual/en/language.oop5.variance.php

Cannot restrict the parameter type.

Reply:
Because that wouldn't be contravariant, i.e. you can only widen the argument type in a child class, not narrow it down.